### ISSUE TITLE
Django 2.2 support

### ISSUE THREAD

**Issue by OskarPersson** [AlexHill/django-relativity#4]
Running the tests illustrates the error perfectly:

```python
ERROR: test_multi_hop (tests.tests.RelationshipTests)

Traceback (most recent call last):
  File "/Users/oskar/git/django-relativity/tests/tests.py", line 302, in test_multi_hop
    self.assertSeqEqual(cart_items, CartItem.objects.filter(product__filters=f))
  File "/Users/oskar/git/django-relativity/tests/tests.py", line 21, in assertSeqEqual
    self.assertSequenceEqual(list(seq1), list(seq2))
  File "/Users/oskar/.pyenv/versions/3.6.7/envs/dj-rel/lib/python3.6/site-packages/django/db/models/query.py", line 274, in __iter__
    self._fetch_all()
  File "/Users/oskar/.pyenv/versions/3.6.7/envs/dj-rel/lib/python3.6/site-packages/django/db/models/query.py", line 1242, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/Users/oskar/.pyenv/versions/3.6.7/envs/dj-rel/lib/python3.6/site-packages/django/db/models/query.py", line 55, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/Users/oskar/.pyenv/versions/3.6.7/envs/dj-rel/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1087, in execute_sql
    sql, params = self.as_sql()
  File "/Users/oskar/.pyenv/versions/3.6.7/envs/dj-rel/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 488, in as_sql
    from_, f_params = self.get_from_clause()
  File "/Users/oskar/.pyenv/versions/3.6.7/envs/dj-rel/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 760, in get_from_clause
    clause_sql, clause_params = self.compile(from_clause)
  File "/Users/oskar/.pyenv/versions/3.6.7/envs/dj-rel/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 405, in compile
    sql, params = node.as_sql(self, self.connection)
  File "/Users/oskar/.pyenv/versions/3.6.7/envs/dj-rel/lib/python3.6/site-packages/django/db/models/sql/datastructures.py", line 86, in as_sql
    extra_sql, extra_params = compiler.compile(extra_cond)
  File "/Users/oskar/.pyenv/versions/3.6.7/envs/dj-rel/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 405, in compile
    sql, params = node.as_sql(self, self.connection)
  File "/Users/oskar/git/django-relativity/relativity/fields.py", line 71, in as_sql
    query=lookup_query, allow_joins=True, reuse=compiler.query.used_aliases
  File "/Users/oskar/.pyenv/versions/3.6.7/envs/dj-rel/lib/python3.6/site-packages/django/db/models/query_utils.py", line 93, in resolve_expression
    clause, joins = query._add_q(self, reuse, allow_joins=allow_joins, split_subq=False)
  File "/Users/oskar/.pyenv/versions/3.6.7/envs/dj-rel/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1318, in _add_q
    split_subq=split_subq, simple_col=simple_col,
  File "/Users/oskar/.pyenv/versions/3.6.7/envs/dj-rel/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1202, in build_filter
    value = self.resolve_lookup_value(value, can_reuse, allow_joins, simple_col)
  File "/Users/oskar/.pyenv/versions/3.6.7/envs/dj-rel/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1025, in resolve_lookup_value
    value = value.resolve_expression(self, **kwargs)
TypeError: resolve_expression() got an unexpected keyword argument 'simple_col'
```

**Comment by AlexHill** [comment]
Resolved by #6

**Comment by OskarPersson** [comment]
Great work @AlexHill  :)
